### DISCUSSION TITLE
milvus-standalone abnormal exit

### DISCUSSION THREAD

**Discussion by ThomasYSJ** [milvus-io/milvus#16017] [Q&A and General discussion]
I execute the code according to this website `https://milvus.io/docs/v2.0.x/install_standalone-docker.md`.
The program started smoothly.
Such as:
      Name                     Command                  State                          Ports
----------------------------------------------------------------------------------------------------------------
milvus-etcd         etcd -listen-peer-urls=htt ...   Up (healthy)   2379/tcp, 2380/tcp
milvus-minio        /usr/bin/docker-entrypoint ...   Up (healthy)   9000/tcp
milvus-standalone   /tini -- milvus run standalone   Up             0.0.0.0:19530->19530/tcp,:::19530->19530/tcp
================================================================================================================
After about half an hour.An error occurred in the process
Such as:
      Name                     Command               State          Ports       
--------------------------------------------------------------------------------
milvus-etcd         etcd -advertise-client-url ...   Up       2379/tcp, 2380/tcp
milvus-minio        /usr/bin/docker-entrypoint ...   Up       9000/tcp          
milvus-standalone   /tini -- milvus run standalone   Exit 0
================================================================================================================
Then I restarted the program.Execute the following command: 
sudo docker-compose down
sudo rm -rf volumes
sudo docker-compose up -d
sudo docker-compose ps

The program started successfully again.
Such as:
      Name                     Command                  State                          Ports
----------------------------------------------------------------------------------------------------------------
milvus-etcd         etcd -listen-peer-urls=htt ...   Up (healthy)   2379/tcp, 2380/tcp
milvus-minio        /usr/bin/docker-entrypoint ...   Up (healthy)   9000/tcp
milvus-standalone   /tini -- milvus run standalone   Up             0.0.0.0:19530->19530/tcp,:::19530->19530/tcp
================================================================================================================
But soon, the same mistake happened.
Such as:
      Name                     Command               State          Ports       
--------------------------------------------------------------------------------
milvus-etcd         etcd -advertise-client-url ...   Up       2379/tcp, 2380/tcp
milvus-minio        /usr/bin/docker-entrypoint ...   Up       9000/tcp          
milvus-standalone   /tini -- milvus run standalone   Exit 0
================================================================================================================
Please help me how to solve the error.Thank you very much!

**Comment by xiaofan-luan** [comment]
hi @ThomasYSJ ，please open an issue with log and that would help us to locate the issue, thanks!

**Reply by VinGPan** [reply]
Hi, I have a similar issue. How to get (where to find) the logfiles?

**Reply by xiaofan-luan** [reply]
Could you open a separate issue with your logs, milvus versions and as much details as possible?
